src/common/translation/i18n/sv.json

```
@@ -480,15 +480,9 @@
       }
     },
     "preliminaryForm": {
-      "childBirthdate": {
+      "childBirthyear": {
         "input": {
-          "day": {
-            "placeholder": "datum"
-          },
-          "label": "Barnets födelsedatum (dd.mm.åååå)",
-          "month": {
-            "placeholder": "mån"
-          },
+          "label": "Barnets födelseår",
           "year": {
             "placeholder": "år"
```

```suggestion
            "placeholder": "Ange barnets födelseår"
```
i.e. "Enter the child's year of birth" in Swedish, making the placeholder more similar to other places in the UI